### ISSUE TITLE
Cartão com a Bandeira trocada

### ISSUE THREAD

**Issue by danilomarcus** [DeveloperCielo/API-3.0-PHP#193]
Prezados, tudo bem? Estou com o seguinte problema:
O cliente escolhe em um SELECT a bandeira como sendo VISA e ele passa na verdade um cartão MASTER e mesmo assim a Cielo aceita e no relatório da cielo aparece como Master, mas o cliente disse que era VISA.

O que será que estou fazendo errado?

Eu fiz um SELECT da seguinte maneira:
`<select class="form-control" name="cc-flag" id="cc-flag">
                                                <option value="Visa">Visa</option>
                                                <option value="Master">Master</option>
                                                <option value="Amex">Amex</option>
                                                <option value="Elo">Elo</option>
                                                <option value="Aura">Aura</option>
                                                <option value="JCB">JCB</option>
                                                <option value="Diners">Diners</option>
                                                <option value="Discover">Discover</option>
                                                <option value="Hipercard">Hipercard</option>
                                                <option value="Hiper">Hiper</option>
                                            </select>`

E no php eu faço da seguinte maneira:

$payment->setType(Payment::PAYMENTTYPE_CREDITCARD)
                ->creditCard($cvv, $cc_flag)
                ->setExpirationDate($cc_expiration)
                ->setCardNumber($cc_number)
                ->setHolder($holder);

O que será que está acontecendo para a Cielo aceitar mesmo passando a Bandeira diferente do que realmente é?
Obrigado!

**Comment by grandalellen** [comment]
Sim, até então cielo não valida isso na transação, deve ser justamente uma jogada para adquirir a  ferramenta como: zero Auth e Consultas BIN que tem justamente essa função de validar o cartão.
